### PR TITLE
OP-583 Fix up Exams List window

### PR DESCRIPTION
**Pull request by dbmalkovsky**
See OP-583

Did a little formatting updating too.

### CODE REVIEW

File: src/main/java/org/isf/lab/gui/LabNew.java [hunk 1]
```diff
@@ -850,7 +851,7 @@ public void actionPerformed(ActionEvent e) {
 					
 					int selectedRow = jTableExams.getSelectedRow();
 					if (selectedRow < 0) {
-						MessageDialog.error(LabNew.this,"angal.labnew.pleaseselectanexam");
+						MessageDialog.error(LabNew.this,"angal.lab.pleaseselectanexam.msg");
```
Comment (mwithi) [inline src/main/java/org/isf/lab/gui/LabNew.java:854]: should we keep only the `angal.lab.` root?

File: src/main/java/org/isf/lab/gui/LabNew.java [hunk 2]
```diff
@@ -781,6 +781,7 @@ public void actionPerformed(ActionEvent e) {
 					examPicker.setSize(300, 400);
 
 					JDialog dialog = new JDialog();
+					dialog.setTitle(MessageBundle.getMessage("angal.stat.examslist")); // TODO: use more correct key
```
Comment (mwithi) [inline src/main/java/org/isf/lab/gui/LabNew.java:784]: What in TODO that cannot be done now?

**Review by mwithi — COMMENTED** [review]
LGTM. Just two comments to understand

File: src/main/java/org/isf/lab/gui/LabNew.java (hunk 2 above)
Comment (dbmalkovsky) [reply src/main/java/org/isf/lab/gui/LabNew.java:784]: See note above; same reasoning.

File: src/main/java/org/isf/lab/gui/LabNew.java (hunk 1 above)
Comment (dbmalkovsky) [reply src/main/java/org/isf/lab/gui/LabNew.java:854]: sure; but I was trying not to make resource bundle changes that might force Transifex to run.

Comment (mwithi) [reply src/main/java/org/isf/lab/gui/LabNew.java:854]: I see

File: src/main/java/org/isf/lab/gui/LabNew.java (hunk 2 above)
Comment (mwithi) [reply src/main/java/org/isf/lab/gui/LabNew.java:784]: Ok

**Pull request merged**